fritzconnection/lib/fritzwlan.py

```diff
@@ -24,17 +24,51 @@
     SEGNO_INSTALLED = True
 
 # important: don't set an extension number here:
-SERVICE = 'WLANConfiguration'
+SERVICE = "WLANConfiguration"
 DEFAULT_PASSWORD_LENGTH = 12
```

Comment:
Changier the quotes from `'` to `"` here and and further style modifications results in a very clustered and hard to understand diff.